bindings/java/src/main/native/bridge.cc

```diff
@@ -4,44 +4,64 @@
  */
 
 #include "bridge.h"
-#include "request_util.h"
+#include "worker.h"
 
 #include <ucp/api/ucp.h>
-#include "ucs/time/time.h"
 
 #include <cstring>
 #include <iostream>
 
 
-#define ERR_EXIT(_msg, _ret)  do {                    \
+#define ERR_EXIT(_msg, _ret)  do {                   \
                                 print_error(_msg);   \
                                 return _ret;         \
-                            } while(0)
+                              } while(0)
 
+#define ERR_JUMP(_msg, _label)  do {                    \
+                                    print_error(_msg);  \
+                                    goto _label;        \
+                                } while(0)
 
-static JavaVM *cached_jvm;
-static jfieldID field_buff = NULL;
-static jfieldID field_addr = NULL;
+static JavaVM   *cached_jvm             = NULL;
+static jfieldID field_comp_queue        = NULL;
+static jfieldID field_worker_addr_arr   = NULL;
 
-static ucp_context_h cached_ctx = NULL;
+static context cached_ctx;
 
 
 static void print_error(const char* error);
 
-// Create context when JNI first loads
-static ucs_status_t create_context();
 
 extern "C" JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM *jvm, void* reserved) {
     cached_jvm = jvm;
     JNIEnv *env;
 
     if (jvm->GetEnv((void **) &env, JNI_VERSION_1_8)) {
-        ERR_EXIT("JNI version 1.8 or higher required.", JNI_ERR);
+        ERR_EXIT("JNI version 1.8 or higher required", JNI_ERR);
     }
 
-    ucs_status_t status = create_context();
-    if (status != UCS_OK) {
-        ERR_EXIT("Failed to create UCP context.", JNI_ERR);
+    jclass queue_cls_data = env->FindClass(
+            "org/ucx/jucx/Worker$CompletionQueue");
+    if (queue_cls_data == NULL) {
+        ERR_EXIT("java org/ucx/jucx/Worker$CompletionQueue class was NOT found",
+                JNI_ERR);
+    }
+
+    field_comp_queue = env->GetFieldID(queue_cls_data, "completionBuff",
+            "Ljava/nio/ByteBuffer;");
```

Comment:
pls indent on column in C and C++ code (in multiple places)

Reply:
Meaning:
```c++
field_comp_queue = env->GetFieldID(queue_cls_data, "completionBuff",
            "Ljava/nio/ByteBuffer;");
```
--->
```c++
field_comp_queue = env->GetFieldID(queue_cls_data, "completionBuff",
                   "Ljava/nio/ByteBuffer;");
```

correct?

Reply:
like this:
```
field_comp_queue = env->GetFieldID(queue_cls_data, "completionBuff",
                                   "Ljava/nio/ByteBuffer;");
```